### ISSUE TITLE
Add option/flag to make usage stats discretionary.

### ISSUE THREAD

**Issue by tecfu** [jsdoc2md/jsdoc-to-markdown#143]
Running this as a grunt job. Tasks fail because jsdoc2md doesn't have permission access the usage directory.

**Comment by 75lb** [comment]
include the `no-usage-stats: true` option in your config, or set `--no-usage-stats` on the command line.

**Comment by tecfu** [comment]
Thanks for the quick reply. Per the options you have documented in lib/jsdoc-to-markdown.js, it appears that I would accomplish this the following way:
```js
    const jsdoc2md = require('jsdoc-to-markdown');
    let jsdocData = jsdoc2md.getTemplateDataSync({
      configure: __dirname + '/conf.json', //jsdoc config file path
      files: 'src/*.js',
    });
    let stdout = jsdoc2md.renderSync({
      data: jsdocData
    });
```

**conf.json**
```
{
  "opts": {
    "no-usage-stats": true
  }
}
```

However, I still get an EACCESS error with this:

```
Error: EACCES: permission denied, open '/home/myuser/.usage-stats/UA-70853320-3-unsent.json'
```

**Comment by 75lb** [comment]
do you have the latest version of jsdoc2md and all its modules installed? I ask because I [fixed this "no write permission" bug](https://github.com/75lb/usage-stats/commit/1f4d36bdf98a10d9bf9b1219f2ac78a127a148de) in the past. 

You pass the flag to `renderSync`, like this: 

```
let stdout = jsdoc2md.renderSync({
  data: jsdocData,
  "no-usage-stats": true
});
```

**Comment by tecfu** [comment]
Thanks for your response.

---

I'm on version 3.0.0.

I also used your snippet with the same result (Error: EACCES: permission denied, open '/home/base/.usage-stats/UA-70853320-3-unsent.json')

```js
    const jsdoc2md = require('jsdoc-to-markdown');
    let jsdocData = jsdoc2md.getTemplateDataSync({
      //configure: __dirname + '/conf.json', //jsdoc config file path
      files: 'src/*.js'
    });
    let stdout = jsdoc2md.renderSync({
      data: jsdocData,
      "no-usage-stats": true
    });
```

**Comment by 75lb** [comment]
I double-checked all the dependencies and discovered one was out of date.. I have released jsdoc2md v3.0.1 - please reinstall. Does this fix your issue?

**Comment by tecfu** [comment]
Thanks for updating your deps. I updated to v3.0.1, but same result. 

I'm not asking you to debug my code, but for reference I'm jsdoc2md as a grunt task here:

https://github.com/tecfu/tty-table/commit/117c83b2084299cccae6bf8f14a4fa504fbf86df

The reason I wanted to disable stats collection is because the jsdoc-to-markdown job throws a permissions error.

**Comment by 75lb** [comment]
Fixed in jsdoc2md v3.0.2 - I have removed usage stats entirely.

**Comment by tecfu** [comment]
Wow. Nice. Thanks!
